lesson_11/test_homework_11.py

```diff
@@ -0,0 +1,175 @@
+
+"""
+Тести для файлу tasks.py
+Запуск: pytest test_tasks.py
+"""
+import pytest
+#import sys
+import math
+#from pathlib import Path
+
+"""
+📝 Завдання 1. Перевірка додавання чисел 
+Напиши тест на функцію add(a, b), яка повертає суму двох чисел. 
+Створи тест, який перевіряє кілька випадків: додавання додатних, від’ємних і нуля.
+"""
+def add(a,b):
+    return a + b
+
+def test_add():
+    assert add(2, 3) == 5
+    assert add(-2, -3) == -5
+    assert add(0, 3) == 3
+    assert add(-5,0) == -5
+
+
+"""
+📝 Завдання 2. Перевірка парності 
+Функція is_even(n) повертає True, якщо число парне, інакше False. 
+Напиши тести для кількох чисел: парних, непарних, від’ємних.
+"""
+def is_even (n):
```

Comment:
```suggestion
def is_even (n):
```
функції треба було імпортувати, не роблять копіювання коду в реалі взагалі ніколи